go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/lightninglabs/neutrino/cache v1.1.1
 	github.com/lightningnetwork/lnd v0.16.0-beta.rc1
 	github.com/lightningnetwork/lnd/cert v1.2.1
+	github.com/lightningnetwork/lnd/clock v1.1.0
```

Comment:
I'm not sure why we're adding `lnd/clock`. What is the goal of this commit?

Reply:
To be able to test sync/proof events that are inserted into the DB with the current timestamp. If we used `time.Now()`, those would only be a few milliseconds apart. But we want to simulate events from multiple days, so we use a clock that we can overwrite. Added this info to the commit message.